packages/selected-network-controller/tests/SelectedNetworkController.test.ts

```diff
@@ -42,12 +42,43 @@ describe('SelectedNetworkController', () => {
       controller.setNetworkClientIdForDomain(domain, networkClientId);
       expect(controller.state.domains[domain]).toBe(networkClientId);
     });
+
+    it('sets the networkClientId for the passed-in domain and updates the provider and block tracker proxy when they already exist', () => {
```

Comment:
This seems to test two pieces of behavior (updating state, and then also updating the proxies). Thoughts on splitting this into two tests?

Reply:
yeah or maybe just delete the first part of the test name / remove one of the expectations